### ISSUE TITLE
Recordings at wrong speed

### ISSUE THREAD

**Issue by GuestJim** [exeldro/obs-source-record#7]
While testing how well this plug-in can be integrated into one of my processes, I've discovered the recordings it produces are notably shorter than the primary recording. More specifically, they seem to be half the length and when playing them in VLC, they appear to run at double speed.

My use-case for these tests is to have a total of four recordings:

- Game recording OBS normally captures
- Overlay 1 (heart-rate monitor)
- Overlay 2 (gaze tracker)
- Composite (Game with both overlays applied)

I have tested twice now and both times the plug-in videos were shorter, though I only looked at the videos from the second test to observe the speed issue.
Here is the relevant OBS log file: [2021-05-25 14-31-14.txt](https://github.com/exeldro/obs-source-record/files/6542061/2021-05-25.14-31-14.txt)

The plug-in videos also lack audio. I have not done any special configuration for their audio, just the audio. Though still a bug, this is not an issue for me as the main recording has all the audio I need.

Definitely looking forward to this issue being fixed, as it would be very nice to record in a single instance of OBS than three and to have a version of the video with the overlays applied.

**Comment by GuestJim** [comment]
Have updated to 0.2.0 and it seems one of its changes has fixed this issue. Not sure about completely trusting it yet, out of caution, but it is presently working exactly as I need and want it to.

**Comment by GuestJim** [comment]
Not sure what happened this time, but with another set of recordings that when for nearly 50 minutes, I found the Source Record files were about a minute shorter than the normal OBS Studio recording. I have done multiple other recordings without this issue, or any issue really, but something is still a little off it appears.
I have attached the log file from that recording.

[2021-06-28 13-32-36.txt](https://github.com/exeldro/obs-source-record/files/6729317/2021-06-28.13-32-36.txt)
